### PR TITLE
Prep for 0.9 Release

### PR DESCRIPTION
**Pull request by blink1073**
- Add test for #50 - note that we cannot test on GitHub Actions because docker is not available on Windows
- Update changelog
- Dropped support for Python 3.4

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ About
 A native Kerberos client implementation for Python on Windows. This module
 mimics the API of `pykerberos <https://pypi.python.org/pypi/pykerberos>`_ to
 implement Kerberos authentication with Microsoft's Security Support Provider
-Interface (SSPI). It supports Python 2.7 and 3.4+.
+Interface (SSPI). It supports Python 3.6+.
```
Comment (behackett) [inline README.rst:13]: Let's not do this yet. Until PyMongo 3 goes EOL in March 2023 we should continue to support Python 2.7 and 3.5 in the modules we maintain that PyMongo depends on. Happily that doesn't really add any burden here.

File: README.rst [hunk 2]
```diff
@@ -29,11 +29,7 @@ Building and installing from source
 You must have the correct version of VC++ installed for your version of
 Python:
 
-- Python 2.7 - `Microsoft Visual C++ Compiler for Python 2.7`_
-- Python 3.4 - Visual Studio 2010 (Professional for 64bit)
-- Python 3.5+ - Visual Studio 2015 (Any version)
-
-.. _`Microsoft Visual C++ Compiler for Python 2.7`: https://www.microsoft.com/en-us/download/details.aspx?id=44266
+- Python 3.6+ - Visual Studio 2015 (Any version)
```
Comment (behackett) [inline README.rst:32]: Add a "+" after 2015. AFAIK any version >= 2015 will work.

**Review by behackett — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (blink1073) [reply README.rst]: Done

**Review by behackett — APPROVED** [review]
LGTM

**Review by ShaneHarvey — APPROVED** [review]

**Comment by blink1073** [comment]
Kicking CI to make sure we're all green

**Pull request merged**